docs/examples-do-k8s-monitoring-infra.md

```diff
@@ -0,0 +1,172 @@
+* [Info](#info)
+* [Requirements](#requirements)
+   * [Set variables regarding to your needs](#set-variables-regarding-to-your-needs)
+      * [Create env file](#create-env-file)
+      * [Customize the enviroment file](#customize-the-enviroment-file)
+      * [Export all variables](#export-all-variables)
+   * [Install terraform](#install-terraform)
+   * [Install cluster.dev](#install-clusterdev)
+   * [Install doctl](#install-doctl)
+   * [Install kubectl](#install-kubectl)
+   * [Create DO's space to store your project's state](#create-dos-space-to-store-your-projects-state)
+   * [Configure DO's API access via token](#configure-dos-api-access-via-token)
+   * [Configure access to the space](#configure-access-to-the-space)
+* [Create basic project](#create-basic-project)
+   * [Customize project](#customize-project)
+* [Create new cluster by the cdev](#create-new-cluster-by-the-cdev)
+* [Destroy the cluster](#destroy-the-cluster)
+* [Known issues](#known-issues)
+   * [Balancer not removed after destroy](#balancer-not-removed-after-destroy)
+   * [You should add your own DNS zone to DO](#you-should-add-your-own-dns-zone-to-do)
+
+# Info
+
+We will learn how to run test Kubernetes cluster in [DigitalOcean](https://www.digitalocean.com/) by [Clusterdev](https://cluster.dev/)
+
+# Requirements
+
+You should have any [Ubuntu 20.04](https://releases.ubuntu.com/20.04/) host to use this manual without any customization
+
+## Set variables regarding to your needs
+### Create env file
+Create default enviroment file:
+```bash
+echo \
+'export PROJECT="example"
+# API access token:
+export DIGITALOCEAN_TOKEN=""
+# Space access token:
+export SPACES_ACCESS_KEY_ID=""
+export SPACES_SECRET_ACCESS_KEY=""
+# Generic default settings:
+export REGION="fra1"
+export BUCKET_NAME="${PROJECT}-cdev-state"
+export ORGANIZATION="${PROJECT}-organization"
+export CLUSTER_NAME="${PROJECT}-cdev"
+export NODE_TYPE="s-4vcpu-8gb-intel"
```

Comment:
This should be defined in stack.yml files itself

Reply:
This also could be configured by Generators https://docs.cluster.dev/generators-overview/

Reply:
My main idea in this document - set all needed settings via env variables, to reduce number of operations to run k8s cluster, which will be used as requirement for any monitoring configuration.
I will try to use generator, probably it will help me with mentioned idea.